mcchris/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>mcchris</title>
+
+		<style>
+			body {
+				background-image: url(./img/bg.jpg);
```

Comment:
`./` is not necessary. If you're going for an absolute path (referencing from the site root), use `/` if not, just go straight for the file name and it will resolve relative to the current file